storage/google/cloud/storage/bucket.py

```diff
@@ -798,10 +798,39 @@ def versioning_enabled(self, value):
         details.
 
         :type value: convertible to boolean
-        :param value: should versioning be anabled for the bucket?
+        :param value: should versioning be enabled for the bucket?
         """
         self._patch_property('versioning', {'enabled': bool(value)})
 
+    @property
+    def requester_pays(self):
+        """Does the requester pay for API requests for this bucket?
+
+        See  https://cloud.google.com/storage/docs/<DOCS-MISSING> for
```

Comment:
I don't object in principle, but in practice we are working on wrapping an API feature with no currently-public docs (like the `pubsub.snapshot` bits).  How would you suggest we address that problem?